wdp/database/ddl.sql

```diff
@@ -1,73 +1,72 @@
 --Creating databse
-CREATE DATABASE wpd_database
+CREATE DATABASE wdp_database
 
 --Creating Employees table
 CREATE TABLE Employees(
-	employee_id INTEGER AUTO_INCREMENT
-	,first_name VARCHAR(50) NOT NULL
-	,last_name VARCHAR(50) NOT NULL
-	,avatar CHAR(50) NOT NULL
-    ,job_title VARCHAR(20)
-	,joined_on DATE NOT NULL
-	,email VARCHAR(50) NOT NULL UNIQUE
-	,phone VARCHAR(12) NOT NULL UNIQUE
-	,birthday DATE NOT NULL
-	,country VARCHAR(50) NOT NULL
-	,city VARCHAR(50) NOT NULL
-	,project_id INTEGER
-	,last_role INTEGER
-	,role_preferred INTEGER NOT NULL
-	,salary FLOAT
-	,specification VARCHAR(50)
-	,PRIMARY KEY (employee_id)
-	,CHECK(joined_on > birthday)
-	,CHECK(birthday < joined_on)
-	,FOREIGN KEY (project_id) REFERENCES Projects(project_id)
-	,FOREIGN KEY (last_role) REFERENCES Roles(role_id)
-	,FOREIGN KEY (role_preferred) REFERENCES Roles(role_id)
+	employee_id INTEGER AUTO_INCREMENT,
+	first_name VARCHAR(250) NOT NULL,
+	last_name VARCHAR(250) NOT NULL,
+	avatar_url VARCHAR(250) NOT NULL,
+    job_title VARCHAR(20),
+	joined_on DATE NOT NULL,
+	email VARCHAR(250) NOT NULL UNIQUE,
+	phone VARCHAR(12) NOT NULL UNIQUE,
+	birthday DATE NOT NULL,
+	country VARCHAR(50) NOT NULL,
+	city VARCHAR(250) NOT NULL,
+	project_id INTEGER,
+	last_role INTEGER,
+	role_preferred INTEGER NOT NULL,
+	salary INTEGER,
+	specification VARCHAR(250),
+	PRIMARY KEY (employee_id),
+	CHECK(joined_on > birthday),
```

Comment:
Kind of a nit-picking of mine: leave a space after the `CHECK` keyword, just to be consistent in spacing.